app/main.py

```diff
@@ -22,12 +24,17 @@ def register_blueprints(app: Flask) -> None:
     """Register `app` blueprints."""
     app.register_blueprint(employee_blueprint)
 
+def configure_jwt(app: Flask) -> None:
+    app.config['JWT_SECRET_KEY'] = settings.SECRET_KEY
```

Comment:
I think there is a JWT_SECRET_KEY in app.core.settings / .env file. Let's use that instead. SECRET_KEY is for the whole api.